drupal/web/themes/custom/drupalcamp_cebu_2018/templates/navigation/menu--hamburger-main-navigation.html.twig

```diff
@@ -0,0 +1,59 @@
+{#
+/**
+ * @file
+ * Theme override to display a menu.
+ *
+ * Available variables:
+ * - menu_name: The machine name of the menu.
+ * - items: A nested list of menu items. Each menu item contains:
+ *   - attributes: HTML attributes for the menu item.
+ *   - below: The menu item child items.
+ *   - title: The menu link title.
+ *   - url: The menu link url, instance of \Drupal\Core\Url
+ *   - localized_options: Menu link localized options.
+ *   - is_expanded: TRUE if the link has visible children within the current
+ *     menu tree.
+ *   - is_collapsed: TRUE if the link has children within the current menu tree
+ *     that are not currently visible.
+ *   - in_active_trail: TRUE if the link is in the active trail.
+ */
+#}
+{% import _self as menus %}
+
+{#
+  We call a macro which calls itself to render the full tree.
+  @see http://twig.sensiolabs.org/doc/tags/macro.html
+#}
+{{ menus.menu_links(items, attributes, 0) }}
+
+{% macro menu_links(items, attributes, menu_level) %}
+  {% import _self as menus %}
+  {% if items %}
+    {% if menu_level == 0 %}
+    <span class="hamburger" onclick="openNav()">&#9776;</span>
+    <div id="side-navigation" class="menu-wrapper">
+      <div class="close-button col-2 col-sm-3 col-md-2"><a href="javascript:void(0)" onclick="closeNav()">&times;</a></div>
+      <ul{{ attributes.addClass('list-group') }}>
+    {% else %}
+      <ul class="list-group">
```

Comment:
Check if we can remove <ul class="list-group"> or not. 
Class list-group is already being added in <ul{{ attributes.addClass('list-group') }}>.